src/App.js

```diff
@@ -0,0 +1,68 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Availability from './components/team/Availability';
+import Splash from './components/authentication/Splash';
+import Login from './components/authentication/Login';
+import Signup from './components/authentication/Signup';
+import Cars from './components/cars/Cars';
+import Details from './components/team/Details';
+import EditUser from './components/authentication/EditUser';
+import Teams from './components/team/Teams';
+import CarCreate from './components/cars/CarCreate';
+import CarDetails from './components/cars/CarDetails';
+import Bookings from './components/bookings/Bookings';
+import Love from './components/team/Love';
+import Quality from './components/team/Quality';
+import Beauty from './components/team/Beauty';
+import Navbar from './components/navbar/Navbar';
+import CarUpdate from './components/cars/CarUpdate';
+import BookingCreate from './components/bookings/BookingCreate';
+import PageNotFound from './components/authentication/PageNotFound';
+
+function App() {
```

Comment:
- [ ] Please, kindly refactor functions and functional components like this  to ES6 arrow functions, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/206780785-992b980e-bc8f-4531-bc07-b0e4b7dae580.png)

Reply:
Hey thank you for that recommendation, That was a default config of the `eslint` formatter. We already fix it.